src/components/map/OlMap.vue

```diff
@@ -702,15 +700,6 @@
         return extent;
       },
 
-      onChangeResolution(event) {
```

Comment:
Why removing this function ? I think it's useful for filtering on `/routes`, `/outings`...

Reply:
As said in #887, this function was added in first YETI PR. It emits a `zoom` event, and no `OLMap` uses it, apart the one inside YETI.

But I can revert it if you want?

Reply:
No perfect, I believed it was used somewhere else, my bad.